src/encoded/schemas/namespaces.json

```diff
@@ -6,6 +6,7 @@
     "GEO": "http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=",
     "Caltech": "http://jumpgate.caltech.edu/library/",
     "FlyBase": "http://flybase.org/cgi-bin/quicksearch_solr.cgi?caller=quicksearch&tab=basic_tab&data_class=FBgn&species=Dmel&search_type=all&context=",
+    "FactorBook": "http://factorbook.beta.wenglab.org/experiment/",
     "WormBase": "http://www.wormbase.org/species/c_elegans/gene/",
```

Comment:
would like to double-check with DAC is the URL not going to change to "non-beta" soon